lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -277,16 +281,30 @@ def test_course_end_date_after_course(self):
     ## Tests Verified Upgrade Deadline Date Block
     @freeze_time('2015-01-02')
     def test_verified_upgrade_deadline_date(self):
-        self.setup_course_and_user(days_till_upgrade_deadline=1)
+        self.setup_course_and_user(days_till_upgrade_deadline=1, user_enrollment_mode=CourseMode.AUDIT)
         block = VerifiedUpgradeDeadlineDate(self.course, self.user)
         self.assertEqual(block.date, datetime.now(utc) + timedelta(days=1))
+        self.assertTrue(block.is_enabled)
         self.assertEqual(block.link, reverse('verify_student_upgrade_and_verify', args=(self.course.id,)))
 
     def test_without_upgrade_deadline(self):
         self.setup_course_and_user(enrollment_mode=None)
         block = VerifiedUpgradeDeadlineDate(self.course, self.user)
+        self.assertFalse(block.is_enabled)
```

Comment:
You should also verify that "Give yourself an additional incentive to complete" is NOT in the resp.content in this case.